docbook/Admin_Guide/en-US/config/issues.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd" [
+<!ENTITY % BOOK_ENTITIES SYSTEM "../Admin_Guide.ent">
+%BOOK_ENTITIES;
+]>
+<section id="admin.config.issues" lang="en-US">
+	<title>Issues visibility</title>
+	 <para>
+		By default, all issues are visible to any user within a project. To limit the visibility of issues there are several mechanisms.
+	</para>
+	 <section id="admin.config.issues.private">
+		<title>Public/Private view status</title>
+		 <para>
+			A view status flag can be set, for an issue, to be either public or private. Private issues are accesible by the user who created it, and by those users that meet a threshold defined in <literal>$g_private_bug_threshold</literal>.
+		</para>
+		 <para>
+			Refer to the following configuration options related to issue view status configurations:
+		</para>
+		 <variablelist>
+			<varlistentry>
+				<term>$g_default_bug_view_status</term>
+				 <listitem>
+					<para>
+						The default viewing status for the new bug (VS_PUBLIC or VS_PRIVATE). The default is VS_PUBLIC.
+					</para>
+
+				</listitem>
+
+			</varlistentry>
+			 <varlistentry>
+				<term>$g_private_bug_threshold</term>
+				 <listitem>
+					<para>
+						The threshold for a user to be able to view any private issue within a project.
+					</para>
+
+				</listitem>
+
+			</varlistentry>
+			 <varlistentry>
+				<term>$g_set_view_status_threshold</term>
+				 <listitem>
+					<para>
+						The threshold for a user to be able to set an issue to Private/Public.
+					</para>
+
+				</listitem>
+
+			</varlistentry>
+			 <varlistentry>
+				<term>$g_change_view_status_threshold</term>
+				 <listitem>
+					<para>
+						The threshold for a user to be able to update the view status while updating an issue.
+					</para>
+
+				</listitem>
+
+			</varlistentry>
+
+		</variablelist>
+
+	</section>
+	 <section id="admin.config.issues.limitreporters">
+		<title>"Limit reporters" configuration (deprecated)</title>
+		 <para>
+			When the option <literal>$g_limit_reporters</literal> is enabled, users that are reporters in a project, or lower access level, can only see the issues they reported. Any issue reporter by other users are not accessibel by them.
```

Comment:
I don't see your version as correct. My reasons:
- "when the option is enabled". The old option was only supported for all projects, not on a per project basis. After this PR, that hasn't changed (becasue the preferred option to be uspported per project, is the new one.)
- "users with REPORTER or lower access". This option evaluates if a user is reporter from the 'report_bug_threshold' option, not by the exact REPORTER access level as defined by name or constants.

Reply:
Right. Reformulating based on your feedback:

```
When the <literal>$g_limit_reporters</literal> option is enabled, users having 
<emphasis>report_bug_threshold</emphasis> access or lower are only 
allowed to see the issues they reported. 
Issues reported by other users are not accessible to them.
```

Reply:
> users having <emphasis>report_bug_threshold</emphasis> access or lower ...

Still not accurate. Because only the lowest of the access levels that meet `report_bug_threshold` is affected by this.
I'd rather leave as it is now, even if simple, as there's the detailed explanation later about the "reporter" term.